ElementsByTagName/feature/tagName14.2.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Tag Name</title>
+</head>
+<body>
+<!doctype html>
+
+<h1>Heading with a <span>span</span> element.</h1>
+<p>A paragraph with <span>one</span>, <span>two</span>
+    spans.</p>
+
+<script>
+    function byTagName(node, tagName) {
+        let found = [];
+        tagName = tagName.toUpperCase();
+
+
+        //What's the name for a function inside of a function?
```

Comment:
A closure -- `explore` is able to "close over" the context where it's defined so you can reference `node`, `tagName` and `found` like you're doing.